### PR TITLE
Postcard

### PR DESCRIPTION
**Pull request by jolf**
Features from the postcard project are ready for review.

### CODE REVIEW

File: src/main/java/dk/kb/aim/workflow/steps/FindFinishedImagesStep.java [hunk 1]
```diff
@@ -29,6 +29,8 @@
     
     /** The name of this step.*/
     protected static final String STEP_NAME = "Find Finished Images step";
+    /** The label for handwriting.*/
+    protected static final String LABEL_HANDWRITING = "Handwriting";
```
Comment (cjen) [inline src/main/java/dk/kb/aim/workflow/steps/FindFinishedImagesStep.java]: protected static final String LABEL_HANDWRITING = "handwriting";

File: db/update_version_2_to_3.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+-- Update from version 2 to version 3, by adding the unique constaint for the text_en and category to the word table.
+ALTER TABLE words
```
[the remark this answers is not in the document]
Comment (danege) [reply db/update_version_2_to_3.sql:2]: AIM-52

File: src/main/java/dk/kb/aim/controller/WordController.java [hunk 3]
```diff
@@ -55,51 +57,73 @@ public String updateWord(@RequestParam("id") int id,
             status = WordStatus.REJECTED;
         }
         String category = parts[1];
-        Word word       = new Word(id, text_en, text_da, category, status);
+        Word word       = new Word(id, text_en.toLowerCase(), text_da.toLowerCase(), category, status);
```
[the remark this answers is not in the document]
Comment (danege) [reply src/main/java/dk/kb/aim/controller/WordController.java:60]: AIM-51

**Review by cjen — APPROVED** [review]

File: src/main/java/dk/kb/aim/Configuration.java [hunk 4]
```diff
@@ -38,6 +38,9 @@
  *   jpeg_url: $ The URL to the image-server where the jpegs can be found.
  *   test: $ ONLY FOR TESTS - will use local images and it will revert the state after finish
  *       instead of finishing. It must be the path to the test-files.
+ *   confidence_limit: $ The lower limit for the confidence of a word belonging to an image. If lower no connection is made
```
Comment (cjen) [inline src/main/java/dk/kb/aim/Configuration.java:41]: Remove "If lower no connection is made"

File: src/main/java/dk/kb/aim/Configuration.java [hunk 5]
```diff
@@ -198,4 +222,23 @@ public File getTestDir() {
     public String getCumulusCatalog() {
         return cumulusConf.getCatalogs().get(0);
     }
+
+    /**
+     * The lower limit for the confidence of a word belonging to an image. If lower than this no connection is made
```
Comment (cjen) [inline src/main/java/dk/kb/aim/Configuration.java:235]: Remove "If lower no connection is made"

**Pull request merged**